paddle/fluid/operators/compat/fused_elementwise_add.pbtxt

```diff
@@ -0,0 +1,50 @@
+type: "fused_elementwise_add"
+def {
+  inputs {
+    name: "X"
+  }
+  inputs {
+    name: "Y"
+  }
+  outputs {
+    name: "Out"
+  }
+  attrs {
+    name: "axis"
+    type: INT
+  }
+}
+extra {
+  attrs {
+    name: "x_data_format"
```

Comment:
`x_data_format` and `y_data_format` seem to be deprecated attributes. Please don't add them here. They should be also removed from base operators